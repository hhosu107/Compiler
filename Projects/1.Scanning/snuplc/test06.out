scanning 'test/test06.mod'...
  1:1: tComment (// \"\\x\")
  2:1: tEOF


Done.
