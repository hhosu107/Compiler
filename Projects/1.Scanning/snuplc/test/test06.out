scanning './test/test06.mod'...
  cannot open input stream: 1:1: tIOError


Done.
